pom.xml

```diff
@@ -123,7 +123,40 @@
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>uk.gov.justice.maven</groupId>
+                <artifactId>annotation-validator-maven-plugin</artifactId>
+                <version>1.0.1</version>
```

Comment:
would be good to have the version as a property

Reply:
done